src/main/java/uk/ac/ebi/tsc/tesk/util/component/KubernetesClientWrapper.java

```diff
@@ -94,6 +98,9 @@ public V1JobList listAllTaskmasterJobsForUser(String pageToken, Integer itemsPer
             // and optionally also to only those jobs, which were created bu the user
             labelSelector += "," + user.getLabelSelector();
         }
+        if(authorisationProperties.isIgnoreGroupMembership()) {
+            labelSelector += "," + new StringJoiner("=").add(LABEL_USERID_KEY).add(user.getUsername()).toString();
```

Comment:
Could you please follow the formatting used in the code (I asked about it before)?